mcpwm_foc.c

```diff
@@ -286,6 +286,38 @@ static volatile bool pid_thd_stop;
 #define M_MOTOR(is_second_motor)  (((void)is_second_motor), &m_motor_1)
 #endif
 
+// This is the noise bit for the PRBS generator
+static bool prbsNoiseBit;
+
+bool mcpwm_foc_get_prbs(void) {
+   return prbsNoiseBit;
+}
+
+/**
+ * @brief prbsGenerator Generates a -1/+1 noise signal from a pseudo-random
+ *        binary sequence with an 11-bit period. Every time this function is
+ *        called, the binary sequences advances one tick.
+ *        C.f. https://blog.kurttomlinson.com/posts/prbs-pseudo-random-binary-sequence
+ *        C.f. https://en.wikipedia.org/wiki/Pseudorandom_binary_sequence
+ *        C.f. https://en.wikipedia.org/wiki/Linear-feedback_shift_register#Example_polynomials_for_maximal_LFSRs
+ * @return -1 or 1
+ */
+int32_t prbsGenerator11_increment(void) {
```

Comment:
`static`

Reply:
I also don't love this being where it is in the code. I feel this should be migrated to `utils.c` or something like that.